### PR TITLE
[ContactStructuralMechanicsApplication] Minor clean up

### PR DESCRIPTION
**Pull request by loumalouomega**
This reduces some code duplication

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp [hunk 1]
```diff
@@ -68,32 +64,29 @@ void InterfacePreprocessCondition::GenerateInterfacePart(
...
-                } else {
-                    GenerateEdgeCondition(rInterfacePart, p_prop, this_geometry, simplest_geometry, cond_counter, cond_id);
+        // We iterate over the elements and check the nodes on the interface
+        for (auto& r_elem : mrMainModelPart.Elements()) {
+            GeometryType& r_geometry = r_elem.GetGeometry();
+            const std::size_t working_space_dimension = r_geometry.WorkingSpaceDimension();
+            const std::size_t local_space_dimension = r_geometry.LocalSpaceDimension();
+
+            Properties::Pointer p_prop = (contact_property_id == 0) ? new_properties[r_elem.pGetProperties()->Id()] : mrMainModelPart.CreateNewProperties(contact_property_id);
+            KRATOS_DEBUG_ERROR_IF(p_prop == nullptr) << "ERROR:: Property not well initialized" << std::endl;
```
Comment (philbucher) [inline applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp:74]:
```suggestion
            KRATOS_DEBUG_ERROR_IF_NOT(p_prop) << "ERROR:: Property not well initialized" << std::endl;
```

**Review by philbucher — APPROVED** [review]

File: applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp:74]: I am not sure about the change..., and the checks already finished

Comment (philbucher) [reply applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp:74]: You can check pointers with true/false if they are valid
Just minor

Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp:74]: I see, but I preferred to compile first, late

**Pull request merged**